src/components/LoansYouMightLike/lymlContainer.vue

```diff
@@ -88,6 +71,10 @@ export default {
 			type: Array,
 			default: () => [],
 		},
+		items: {
```

Comment:
I think this is unused and can be removed.

Reply:
✔️